.github/workflows/build.yaml

```diff
@@ -43,17 +47,12 @@ jobs:
           git fetch --prune --unshallow
       - name: Define version
         id: version
-        env:
-          hash: ${{ hashFiles('Dockerfile', '**/go.sum', '**/pkg/**', '**/examples/**', '**/cmd/**', '**/vendor/**', '**/Makefile', '**/main.go') }}
-        run: |
-          version="${{ env.hash }}"
-          version=${version::16}
-          echo "version=${version}" >> $GITHUB_OUTPUT
+        uses: ./.github/actions/version
```

Comment:
Moved to an action file as this is used twice and the paths used for the need to be exactly the same. So to avoid misalignment and duplication this logic has been moved to `.github/actions/version`